nixos/modules/services/networking/wireguard.nix

```diff
@@ -246,12 +246,15 @@ let
         };
 
         script = ''
-          mkdir --mode 0644 -p "${dirOf values.privateKeyFile}"
+          set -e
```

Comment:
This is automatic:

https://github.com/NixOS/nixpkgs/blob/c709fd42d9f7b7df45dd730dbe5b56817dfe0e5b/nixos/modules/system/boot/systemd.nix#L213

Reply:
Yes, but who knows if it will stay that way forever; thus especially for the more sensitive stuff, it may be good to be explicit.

(Also evidently a ton of shell users don't know about `set -e`, that it's off by default, and on-by-default in _some_ NixOS scripts, so they may get surprises when copying scripts from one place to the other.)